velox/functions/sparksql/Arithmetic.h

```diff
@@ -368,6 +368,68 @@ FOLLY_ALWAYS_INLINE static int8_t fromHex(char c) {
 }
 } // namespace detail
 
+template <typename T>
+struct WidthBucketFunction {
+  FOLLY_ALWAYS_INLINE bool call(
+      int64_t& result,
+      double value,
+      double min,
+      double max,
```

Comment:
These names are confusing because sometimes min > max.

Reply:
updated to `bound1` and `bound2`